pinkopy/base_session.py

```diff
@@ -56,23 +57,23 @@ def __init__(self, service, user, pw, use_cache=True, cache_ttl=1200,
                 self.__enable_method_cache(method_name)
 
     def __enable_method_cache(self, method_name):
-        """Enable cache for a method.
+        """
+        Enable cache for a method.
```

Comment:
This was following the [Google Styleguide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).